Sources/libtkey/include/tkey.h

```diff
@@ -80,6 +79,7 @@
         void generate_share_store_result_free(struct GenerateShareStoreResult* ptr);
         void share_store_poly_id_index_map_free(struct ShareStorePolyIDShareIndexMap* ptr);
         struct GenerateShareStoreResult* threshold_key_generate_share(struct FFIThresholdKey* threshold_key, char* curve_n, bool use_tss, struct TssOptions* tss_options, int* error_code);
+        void threshold_key_import_tss_key(struct FFIThresholdKey* threshold_key, bool update_metadata, char* tss_tag, char* import_key, int new_tss_index, struct KeyPoint* factor_pub, char* selected_servers, char* auth_signatures, char* curve_n, int* error_code);
```

Comment:
We should implement these when they get added. Doesn't need to be in this PR though since it is unrelated.

Can we also just double check the version and package name in this repository, since it was split away from the original?

Otherwise looks good.